pageserver/src/tenant.rs

```diff
@@ -1253,8 +1166,8 @@ impl Tenant {
                             LoadLocalTimelineError::Load(source) => {
                                 // We tried to load deleted timeline, this is a bug.
                                 return Err(anyhow::anyhow!(source).context(
-                                "This is a bug. We tried to load deleted timeline which is wrong and loading failed. Timeline: {timeline_id}"
-                            ));
+                                    "This is a bug. We tried to load deleted timeline which is wrong and loading failed. Timeline: {timeline_id}"
+                                ));
```

Comment:
Well well well, this is also a missing format string.

Reply:
Want to address this?

Reply:
I'll do it.